src/util/misc/objectTransforms.ts

```diff
@@ -0,0 +1,91 @@
+import { Point } from "../../point.class";
+import { TMat2D } from "../../typedefs";
+import { invertTransform, multiplyTransformMatrices, qrDecompose } from "./matrix";
+import type { TComposeMatrixArgs } from './matrix';
+
+type FabricObject = any;
```

Comment:
Maybe we should put this in typedefs so it will be easy to replace...
I didn't put too much thought into it when I did it

Reply:
me neither. is just to remember we need a generic type that accept any fabricObject interface.